### PR TITLE
chore: playground equipment: lint / test dev tools

### PR DESCRIPTION
**Pull request by dckc**
factored out of #2

question: should packages such as ag-trade have their own yarn.lock? Or should the whole playground share a yarn.lock?

### CODE REVIEW

**Comment by kriskowal** [comment]
I like the idea of separate lockfiles for various experiments.

File: packages/ag-trade/CHANGELOG.md
Comment (kriskowal) [inline packages/ag-trade/CHANGELOG.md:1]: can be deleted

File: packages/ag-trade/node_modules/.bin/ava
Comment (kriskowal) [inline packages/ag-trade/node_modules/.bin/ava:1]: shouldn’t be checked in. maybe need .gitignore.

File: packages/ag-trade/package.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@endo/skel",
```
Comment (kriskowal) [inline packages/ag-trade/package.json:2]: looks like the skeleton unaltered

**Review by kriskowal — APPROVED** [review]
Don’t trouble to make this perfect.

I don’t feel strongly about whether this should closely resemble the endo repository, but copying `scripts` and `packages/skel` along with all the lint configuration you have here certainly makes a lot easier.

Not afraid of a mess in this repo. Maybe someday some stuff graduates to an analogous `@endo/kit` repo.

File: packages/ag-trade/package.json (hunk 1 above)
Comment (dckc) [reply packages/ag-trade/package.json:2]: yes, this is exactly the skeleton, unaltered; i.e. before we start playing

I'm a little surprised that there are any `ag-trade/` files here at all, but I suppose the way I did this was to run the skeleton widget and then try to test / lint and add equipment as needed

File: packages/ag-trade/node_modules/.bin/ava
Comment (dckc) [reply packages/ag-trade/node_modules/.bin/ava:1]: do you know how it got there at all?

**Pull request merged**